### PR TITLE
DOC: updates to two cookbook recipes

### PR DESCRIPTION
**Pull request by dmci99**
Some edits to documentation in the cookbook.

### CODE REVIEW

File: Documentation/Cookbook/rst/recipes/bandmathx.rst [hunk 1]
```diff
@@ -99,19 +97,18 @@ represents a pixel of an image made of only one band:
 
 .. math:: im1+1
 
-A scalar can’t be added to a vector. The right formula is instead (one
-can notice the way that muParserX allows to define vectors on the fly):
+A scalar can’t be added to a vector, therefore the correct syntax for the formulta is as follows (one
```
Comment (gpo-geo) [inline Documentation/Cookbook/rst/recipes/bandmathx.rst]: typo : formula

File: Documentation/Cookbook/rst/recipes/improc.rst [hunk 2]
```diff
@@ -195,22 +194,21 @@ its value, and their labels will be set to 0 (nodata).
...
-sequentially with respect to the order in which tiles are processed. You
-will see after the result of the vectorization step that there are no
-artifacts in the results.
+sequentially with respect to the order in which tiles are processed. However, 
+all of the artefacts will be removed after the vectorization setp. 
 
 The *LSMSSegmentation* application will write as many intermediate files
-as tiles needed during processing. As such, it may require twice as free
-disk space as the final size of the final image. The *cleanup* option
+as tiles needed during processing. As such, it will sufficient disk space, typically
```
Comment (gpo-geo) [inline Documentation/Cookbook/rst/recipes/improc.rst]: ... will require sufficient ...

File: Documentation/Cookbook/rst/recipes/improc.rst [hunk 3]
```diff
@@ -245,8 +243,8 @@ one polygon per segment, and each of these polygons will hold additional
 attributes denoting the label of the original segment, the size of the
 segment in pixels, and the mean and variance of each band over the
 segment. The projection of the output GIS vector file will be the same
-as the projection from the input image (if input image has no
-projection, so does the output GIS file).
+as the projection from the input image (if the input image does not have a
+projection, than the output GIS files won't either).
```
Comment (gpo-geo) [inline Documentation/Cookbook/rst/recipes/improc.rst]: than ->then ?

**Comment by gpo-geo** [comment]
... or if you have push access to official repo, you can do the merge (see indications [here](https://wiki.orfeo-toolbox.org/index.php/Git#If_you_have_a_Pull_request_on_github.com.2Forfeotoolbox.2FOTBl) )

**Comment by grizonnetm** [comment]
Daniel is an official committer so he could indeed have access to the official repository. I did not create the git account yet as I wanted to see how the discussion about migrating OTB process to github goes. I imagine that it will be discuss in the next psc meeting of Tursday.

**Pull request closed without merging**